bin/update-hoc-map

```diff
@@ -39,7 +39,8 @@ CSV.open(file.path, 'w') do |csv|
   all_forms.each do |form|
     data = JSON.parse(form[:data])
     processed_data = JSON.parse(form[:processed_data])
-    next if data['organization_name_s'].nil? || processed_data['location_city_s'].nil? || processed_data['location_p'].nil?
+    next if data['organization_name_s'].nil? || processed_data['location_p'].nil?
+    next if form[:review] != "approved" && processed_data['location_city_s'].nil?
```

Comment:
how about a comment so whoever is updating this next year knows what this means?